explorer/src/pages/TransactionDetailsPage.tsx

```diff
@@ -67,9 +74,13 @@ function StatusCard({ signature }: Props) {
   const fetchDetails = useFetchTransactionDetails();
   const details = useTransactionDetails(signature);
   const { firstAvailableBlock, status: clusterStatus } = useCluster();
+  const [autoRefresh, setAutoRefresh] = React.useState<AutoRefresh>({
+    isAutoRefresh: false,
+  });
+
   const refresh = React.useCallback(
-    (signature: string) => {
-      fetchStatus(signature);
+    (signature: string, isAutoRefresh: boolean = false) => {
+      fetchStatus(signature, isAutoRefresh);
```

Comment:
I think auto refresh state can stay local to this component. I don't see a need to add it to the context provider